apps/browser-tests/src/App.tsx

```diff
@@ -1,12 +1,20 @@
 import { useCallback, useMemo, useState } from "react";
 import { Address, PSBTBuilder, PSBTBuilderOptions } from "@ordzaar/ordit-sdk";
+import * as magiceden from "@ordzaar/ordit-sdk/magiceden";
 import * as unisat from "@ordzaar/ordit-sdk/unisat";
 import * as xverse from "@ordzaar/ordit-sdk/xverse";
 
 import { RadioInput } from "./components/RadioInput";
 import { Select } from "./components/Select";
 
-type WalletProvider = "unisat" | "xverse";
+type WalletProvider = "unisat" | "xverse" | "magiceden";
+
+const TESTNET = "testnet" as const;
+// const MAINNET = "mainnet" as const;
```

Comment:
For another ticket @danadi7, can just add a button here instead of hard coding it. Not for this PR

Reply:
This was intentional, I didn't want to make it too easy to switch to `mainnet` for safety reasons just in case.

Reply:
But this is only a testing app :D